channels/generic/http.py

```diff
@@ -1,8 +1,21 @@
+import logging
+import traceback
+
 from channels.consumer import AsyncConsumer
 
 from ..db import aclose_old_connections
 from ..exceptions import StopConsumer
 
+logger = logging.getLogger("channels.consumer")
+if not logger.hasHandlers():
+    handler = logging.StreamHandler()
+    formatter = logging.Formatter(
+        "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
+    )
+    handler.setFormatter(formatter)
+    logger.addHandler(handler)
+    logger.setLevel(logging.DEBUG)
+
```

Comment:
Adding the logging call is fine I guess, but I don't think we should be adding handlers for users. That's up to them.